plugins/hotfix.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: hotfix
+spec:
+  version: "v0.0.3"
+  shortDescription: dumb file transfer to your labelled pods
+  homepage: https://steady.supply/git/kubectl-hotfix
+  description: |
+
+      hotfix
+
+      dumb file sync/transfer to your labelled pods
+
+      command line options are -v, etc for glog, and -c to specify a configuration
+      file, eg.
+
+        namespace: xyz
+        ignore:
+          - "*.tmp"
+          - "*.{html,css,js}"
+        include:
+          - "*.py"
+        shell: ash
+        path: src:/app
+        target:
+          - label: service in (web, api)
+          - label: service=postgres
+            path: src/database:/app/database
+          - label: service=chrome
+            shell: bash
+          - label: service=static
+            include:
+              - "*.{html,css,js}"
```

Comment:
This should be moved into a dedicated help for this command like `hotfix --help`. The description is meant to explain _what_ a plugin does and not how to configure it.